src/app/shared/validators/age-range-validator.ts

```
@@ -0,0 +1,32 @@
+    if (startCtrl.value && endCtrl.value && startCtrl.value > endCtrl.value) {
+      const newGroupErrors = { invalidAgeRange: true };
+
+      startCtrl.setErrors({ ...startCtrl.errors, invalidAgeRange: true });
+      endCtrl.setErrors({ ...endCtrl.errors, invalidAgeRange: true });
+
+      return newGroupErrors;
+    } else {
+      const startErrors = startCtrl.errors || {};
+      const endErrors = endCtrl.errors || {};
+
+      delete startErrors?.invalidAgeRange;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)